### ISSUE TITLE
Error in current quota

### ISSUE THREAD

**Issue by ajcanepa** [samterfa/openai#8]
After following the installation and set-up steps ([here](https://github.com/samterfa/openai#set-up)), when I click on the XXX, I received this message:
`Error in (function(prompt=rstudioapi::getConsoleEditorContext()$contents,:list(message = "You exceeded your current quota, please check your plan billing details", type = "insuficient_quota", param = NULL, code = NULL)`...

I was wondering if is mandatory to pay to use the API, is that the main problem?

Thanks!

**Comment by samterfa** [comment]
Yes, unfortunately that is the case. I need to add that step to the readme.
